web-marketplace/src/pages/Projects/Projects.SideFilter.tsx

```diff
@@ -115,12 +118,18 @@ export const ProjectsSideFilter = ({
                             checked={
                               creditClassSelectedFilters?.[path] ?? false
                             }
-                            onChange={event =>
+                            onChange={event => {
                               setCreditClassFilter({
                                 ...creditClassSelectedFilters,
                                 [path]: event.target.checked,
-                              })
-                            }
+                              });
+                              track<
+                                'filterCreditClass',
+                                FilterCreditClassEvent
+                              >('filterCreditClass', {
+                                creditClassId: path,
```

Comment:
asked @erikalogie whether we should also specify whether it's about selecting or unselecting some credit class

Reply:
updated based on latest requirements